core/src/apps/base.py

```diff
@@ -75,6 +75,11 @@ def get_features() -> Features:
             Capability.ShamirGroups,
             Capability.PassphraseEntry,
         ]
+
+    # Model R is not capable of PassphraseEntry
+    if utils.MODEL == "R":
+        f.capabilities.remove(Capability.PassphraseEntry)
+
```

Comment:
This is not very nice. How about removing `PassPhraseEntry` from two lists above and adding the following?

``` python
if utils.MODEL == "T":
    f.capabilities.append(Capability.PassphraseEntry);
```